src/main/webapp/content/scss/quiz.scss

```diff
@@ -1190,3 +1190,70 @@ table.answer-options-result {
     visibility: visible;
     opacity: 1;
 }
+
+/* StepwizardQuiz for the progress bar*/
+
+.stepwizardquiz-step p {
+    margin-top: 10px;
+}
+
+.stepwizardquiz-step .fa {
+    vertical-align: initial;
+}
+
+.stepwizardquiz-row {
+    display: table-row;
+}
+
+.stepwizardquiz {
```

Comment:
can we define a max width for this Indicator (possibly using the SASS Variables defined by bootstrap) and do a line break if there are too many steps.
Alternatively we could also use horizontal scrolling instead.
Elements should not overlap!

Reply:
I like the idea of a max-width with a line break in case we have too many elements.